push/templates/push/validate.html

```diff
@@ -0,0 +1,19 @@
+{% extends "dashboard/base.html" %}
+
+{% block content %}
+    {% if not request.user.is_authenticated %}
+        {% include 'dashboard/includes/authenticate.html' %}
```

Comment:
Don't burn yourself with too much time on it; happy to file an issue for next PR/merge/deploy.